gradle-enterprise-conventions-gradle-plugin/plugins/gradle-2-through-4/src/main/java/com/myorg/BuildScansConventionsPlugin.java

```diff
@@ -0,0 +1,24 @@
+package com.myorg;
+
+import com.gradle.scan.plugin.BuildScanExtension;
+import com.gradle.scan.plugin.BuildScanPlugin;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BuildScansConventionsPlugin implements Plugin<Project> {
```

Comment:
Maybe worth some explanatory class level Javadoc explaining that this is the plugin used for < Gradle 5 etc.

Reply:
Added